### PR TITLE
nextlevel-crimelife

### PR DESCRIPTION
**Pull request by zShynox**
## Type of change

- [x] Add new directory for a server
- [ ] Update directory of a server
- [ ] Remove directory of a server
- [ ] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/nextlevel-crimelife/manifest.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+  "server_name": "nextlevel-crimelife",
+  "nice_name": "Nextlevel-Crimelife",
+  "direct_ip": "nextlevel-crimelife.net",
+  "server_wildcards": [
+    "%.nextlevel-crimelife.net"
```
Comment (maaaathis) [inline minecraft_servers/nextlevel-crimelife/manifest.json:6]: Doesn't look like a real wildcard
image

**Review by maaaathis — CHANGES_REQUESTED** [review]

File: minecraft_servers/nextlevel-crimelife/manifest.json (hunk 1 above)
Comment (zShynox) [reply minecraft_servers/nextlevel-crimelife/manifest.json:6]: now it should work

**Comment by DoJapHD** [comment]
Is the server already released?

**Comment by zShynox** [comment]
im working on it to release it soon

**Comment by DoJapHD** [comment]
Server needs to be already released;
Reopen PR once server is available to public

**Pull request closed without merging**